relentless/simulate/lammps.py

```diff
@@ -512,107 +582,135 @@ def to_commands(self, sim):
 
         return cmds
 
-class AddNPTIntegrator(LAMMPSOperation):
-    r"""NPT integration via Nos\'e-Hoover thermostat/barostat.
+class AddVerletIntegrator(LAMMPSOperation):
+    """Family of Verlet integration modes.
+
+    This method supports:
+
+    - NVE integration with optional Berendsen thermostat and optional Berendsen barostat
+    - NVT integration with Nosé-Hoover thermostat and optional Berendsen barostat
+    - NPH integration with MTK barostat and optional Berendsen thermostat
+    - NPT integration with Nosé-Hoover thermostat and MTK barostat
```

Comment:
I think this list is cleaner, did I get them all right?
```suggestion
    - NVE integration
    - NVT integration with Nosé-Hoover or Berendsen thermostat
    - NPH integration with MTK or Berendsen barostat
    - NPT integration with  Nosé-Hoover or Berendsen thermostat and MTK or Berendsen barostat
```